### PR TITLE
Poking a clown mob produces the proper response.

### PR DESCRIPTION
**Pull request by harmonyn**
As does hitting a clown, etc.

Some verbs and text also changed to be more clownlike.

### CODE REVIEW

**Comment by KazeEspada** [comment]
It would be nice if the first clown struck unleashed a mighty airhorn to rally nearby clowns. Probably difficult to program without causing a ripple honk.

**Comment by bgobandit** [comment]
merge when

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm [hunk 1]
```diff
@@ -36,3 +36,7 @@
 		adjustBruteLoss(10)
 	else if(bodytemperature > maxbodytemp)
 		adjustBruteLoss(15)
+
+/mob/living/simple_animal/hostile/retaliate/clown/attack_hand(mob/living/carbon/human/M)
```
Comment (tkdrg) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: This happening only on attack_hand and not attackby etc looks weird. Maybe make it generic (something like the hit_reaction system in /human). Just make simple_animals call some proc when they get hit, and call that.

Comment (bgobandit) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: do help intent/etc count as hits for those

Comment (tkdrg) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: i think so but i'm not sure

Comment (bgobandit) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: if not then that kind of defeats the purpose of the PR

Comment (phil235) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: yes, help intent touching does call attack_hand so it will play the sound.

Comment (Cheridan) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: ok so this is good to merge then??

**Comment by Lumipharon** [comment]
>It would be nice if the first clown struck unleashed a mighty airhorn to rally nearby clowns. Probably difficult to program without causing a ripple honk.

You make that sound like a bad thing.
It would be a blood curdling war cry, shit would be neat.

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm (hunk 1 above)
Comment (tkdrg) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm]: atm it only works on help intent and does nothing on disarm/harm, making it inconsistent with human clowns

**Pull request merged**